bioindex/src/main/scala/VariantsStage.scala

```diff
@@ -24,8 +19,14 @@ class VariantsStage(implicit context: Context) extends Stage {
     case common() => Outputs.Named("variants")
   }
 
+  override val cluster: ClusterDef = {
+    super.cluster.copy(
+      releaseLabel = ReleaseLabel("emr-6.7.0")
+    )
+  }
+
```

Comment:
Figured we should upgrade to more current emr.  I defer to you on proper cluster settings for this job.